etc/ci/buildbot_steps.yml

```diff
@@ -23,8 +23,8 @@ mac-rel-css:
   - bash ./etc/ci/manifest_changed.sh
 
 linux-dev:
```

Comment:
I swapped the order of the `test-tidy`s so that it matched the order in `travis.yml`.